src/logger.ts

```diff
@@ -1,9 +1,9 @@
-import Pino from 'pino'
-import { getLevel, isInstalled } from './util'
+import { Logger, LoggerOptions, pino } from 'pino'
```

Comment:
This is preferred over default exports. I was seeing the same error as in this issue: https://github.com/pinojs/pino/issues/1452#issuecomment-1153335237